### ISSUE TITLE
imagenet/multiedge_inference_bench: save_weights() stores PyTorch tensors instead of numpy arrays in bert.py

### ISSUE THREAD

**Issue by Nityahiray** [kubeedge/ianvs#478]
**What happened**:
In bert.py, the save_weights() function in both 
BertModelShard and BertShardForSequenceClassification 
stores PyTorch tensor objects directly into a dictionary 
and then calls np.savez() to save them. But np.savez() 
only supports numpy arrays, not PyTorch tensors. 
This crashes with a TypeError when anyone tries to 
save model weights.

**What you expected to happen**:
The model weights should be saved successfully as a 
.npz file without any errors.

**How to reproduce it (as minimally and precisely as possible)**:
1. Go to examples/imagenet/multiedge_inference_bench
2. Run BertModelShard.save_weights() or 
   BertShardForSequenceClassification.save_weights()
3. The program crashes with TypeError

**Anything else we need to know?**:
Fix: Change the line to:
weights[key] = val.detach().cpu().numpy()

This bug exists in both:
- BertModelShard.save_weights()
- BertShardForSequenceClassification.save_weights()
